tests/metrics_tests.cpp

```diff
@@ -99,7 +99,15 @@ class MetricsTest : public MesosTest
           } ]
         }
         )~",
+
+        // TODO(tillt): We need to rework this for Windows as soon as
+        // there are standalone modules on that platform.
+#ifndef __WINDOWS__
         path::join(MODULES_BUILD_DIR, ".libs", "libmetrics-module.so"),
+#else
+        "C:/fake/location/metrics-module.dll",
+#endif // __WINDOWS__
+
```

Comment:
I was wondering why this line would not work for this test, whereas other usages of `MODULES_BUILD_DIR` appear fine.  The other tests tend to use `MODULES_BUILD_DIR` to read a templated `modules.json` file.  `os::read` will handle with mixed separators just fine (`MODULES_BUILD_DIR` is generated by CMake with `/`).

We don't have other tests inserting the module library into the module protobuf.

Reply:
Presumably, there is a JSON parsing error because inserting `MODULES_BUILD_DIR\.libs\libmetrics-module.so` into the JSON includes un-escaped backslashes.